src/main/java/net/minestom/server/entity/metadata/AgeableMeta.java

```diff
@@ -0,0 +1,12 @@
+package net.minestom.server.entity.metadata;
+
+public interface AgeableMeta {
+    boolean isBaby();
+
+    /**
+     * Forcefully sets the bounding box of this entity to the bounding box
+     * which the client expects.
```

Comment:
Javadocs seem somewhat incomplete? It should also mention that it changes the entity's baby status, if I looked at this method and it only said that it changed the bounding box I'd be a bit confused